eng/pipelines/runtime.yml

```diff
@@ -500,6 +500,20 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
+    platforms:
+    - Linux_x64
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
```

Comment:
You don't need `helixQueuesTemplate` that will just add an extra layer in the yml expansion and that is only needed for when we run tests.